front/app/components/ReactionControl/ReactionButton.tsx

```diff
@@ -404,19 +404,15 @@ const ReactionButton = ({
     const projectName = localize(project.data.attributes.title_multiloc);
     const buttonReactionModeIsActive = buttonReactionMode === userReactionMode;
 
-    const describedById = `tooltip-${ideaId}`;
+    const describedById = disabledReason ? `tooltip-${ideaId}` : undefined;
 
     const disabledMessage = disabledReasonMessage && (
-      // We can't put id on FormattedMessage:
-      // - If it's after the message descriptor, it'll overwrite the message descriptor's id.
-      // - It it's before the message descriptor, it'll be overwritten by the message descriptor's id.
       <FormattedMessage
         {...disabledReasonMessage}
         values={{
           enabledFromDate,
           projectName,
         }}
-        aria-hidden
```

Comment:
This doesn't get passed through via `FormattedMessage`. Also, we wouldn't want it on our `disabledMessage` since `aria-hidden` would then also be put on the content of `ScreenReaderOnly` (which also uses `disabledMessage`).

Reply:
We need to wrap it at the point where we pass it to `Tooltip` (see changes on the right).